src/sentry/static/sentry/app/components/customResolutionModal.tsx

```diff
@@ -18,12 +32,11 @@ export default class CustomResolutionModal extends React.Component {
     show: PropTypes.bool,
   };
 
-  constructor(...args) {
-    super(...args);
-    this.state = {version: ''};
-  }
+  state = {
+    version: '',
+  };
 
-  componentDidUpdate(prevProps, prevState) {
+  componentDidUpdate(prevProps) {
     if (!prevProps.show && this.props.show) {
       // XXX(cramer): this is incorrect but idgaf
       $('.modal').attr('tabindex', null);
```

Comment:
Classic cramer code right here. Any chance we could kill jquery while we're in here?

Reply:
I played with it a little and I think we can safely kill the whole componenDidUpdate. Modal seems to be smart enough with tabindex.